packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -927,6 +802,22 @@ export class SDKClient {
         `${formattedRequestId} Fail to executeAll for ${transactionType} transaction: transactionId=${transaction.transactionId}, callerName=${callerName}, transactionType=${transactionType}, status=${sdkClientError.status}(${sdkClientError.status._code})`,
       );
       throw sdkClientError;
+    } finally {
+      // Loop through transactionResponses to asynchronously capture metrics from each response
```

Comment:
No need to explain what is already self-explanatory from the code, please remove these redundant comments. 

Comments should explain "why" instead of "what".

Reply:
yeah good points the comments were meant to be a todo list for me during programming. Removed in new updates.